### PR TITLE
Docker diagnostics

### PR DESCRIPTION
**Pull request by psiinon**
Depends on https://github.com/zaproxy/community-scripts/pull/276

Screenshot 2022-03-24 at 16-33-36 OWASP ZAP – Diagnosing Docker Problems

I took the screenshot before rebasing, hense the ZAPCon banner ;)

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/docker/diagnosing-problems.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+title: "Diagnosing Docker Problems"
+tags: 
+- docker
+- guide
+type: docker
+---
+
+Docker is a great way to run ZAP in a CI/CD pipeline, but diagnosing problems can be tricky.
+
+## ZAP Cannot Connect To The Target
```
Comment (kingthorin) [inline site/content/docs/docker/diagnosing-problems.md:11]:
```suggestion
## ZAP Cannot Connect to the Target
```

File: site/content/docs/docker/diagnosing-problems.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+If ZAP cannot connect to your target app then the first thing to do is to see if this is a 'Docker' networking issue.
```
Comment (kingthorin) [inline site/content/docs/docker/diagnosing-problems.md:13]: Ha! You fixed it as I was marking it up 😀

**Review by thc202 — APPROVED** [review]

**Comment by psiinon** [comment]
Done :)

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by psiinon** [comment]
Pushed with no changes to unblock the checks...

**Pull request merged**